### ISSUE TITLE
Define global config

### ISSUE THREAD

**Issue by celsomtrindade** [KillerCodeMonkey/ngx-quill#171]
I was wondering if is there a way to declare global config. I'm importing the `quill-editor ` in different views but all of them share the same type of configuration. So instead of declaring a local variable to insert into the modules, is there a way to declare it globally when importing the module?

For example, instead of

```
export class EmpresaComponent {
    public modules = toolbar [.....];
}
```

Do something like:

```
imports: [ QuillModule ],
providers: [ 
    { provide: QuillEditorComponent, useClass: QuillConfig }
],
```

**Comment by KillerCodeMonkey** [comment]
Süße, Feel free to send me a PR.

I thought about a Service which holds the default config

Celso Trindade <notifications@github.com> schrieb am Fr., 4. Mai 2018,
14:59:

**Comment by onerzafer** [comment]
@KillerCodeMonkey Do you have any plan for this feature to be released ?

**Comment by KillerCodeMonkey** [comment]
maybe this evening in central european time ;)

**Comment by KillerCodeMonkey** [comment]
released in v4.2.0.

now you can import `QuillModule` as 

```
QuillModule

QuillModule.forRoot()

QuillModule.forRoot({
  modules: {
    toolbar: [...]
  }
})
```

if nothing is passed to the `forRoot`-function the default toolbar config is used.
